src/it/verify-it/verify.groovy

```diff
@@ -31,6 +35,34 @@ assert new File(basedir, 'target/generated-sources/localizer/org/jenkinsci/tools
 content = new File(basedir, 'target/generated-sources/localizer/org/jenkinsci/tools/hpi/its/Messages.java').text;
 assert content.contains(" holder.format(\"it.msg\");");
 
+assert new File(basedir, 'target/verify-it/META-INF/MANIFEST.MF').exists()
+
+InputStream is = Files.newInputStream(new File(basedir, 'target/verify-it/META-INF/MANIFEST.MF').toPath())
+try {
+  Manifest manifest = new Manifest(is)
+  assert !manifest.getMainAttributes().getValue('Build-Jdk-Spec').isEmpty()
+  assert manifest.getMainAttributes().getValue('Created-By').startsWith('Maven Archiver')
+  assert manifest.getMainAttributes().getValue('Extension-Name') == null // was provided by Maven 2, but core prefers Short-Name
+  assert manifest.getMainAttributes().getValue('Group-Id').equals('org.jenkins-ci.tools.hpi.its')
+  assert manifest.getMainAttributes().getValue('Hudson-Version').equals('2.249.1')
+  assert manifest.getMainAttributes().getValue('Implementation-Title').equals('MyNewPlugin') // was project.artifactId in previous versions, now project.name
```

Comment:
Maven 2 set this to the artifact ID, but Maven 3 sets this to the project name. I couldn't find any code in the ecosystem that relied on this being set to any particular value.